src/FSharpLint.Core/Rules/Conventions/NoPartialFunctions.fs

```diff
@@ -96,13 +110,92 @@ let private checkIfPartialIdentifier (config:Config) (identifier:string) (range:
                     TypeChecks = []
                 })
 
+let private isNonStaticInstanceMemberCall (checkFile:FSharpCheckFileResults) names range:(Option<WarningDetails>) =
+
+    let typeChecks =
+        (partialInstanceMemberIdentifiers
+        |> Map.toList
+        |> List.map (fun replacement ->
+            match replacement with
+            | (fullyQualifiedInstanceMember, replacementStrategy) ->
+                if not (fullyQualifiedInstanceMember.Contains ".") then
+                    failwith "Please use fully qualified name for the instance member"
+                let nameSegments = fullyQualifiedInstanceMember.Split '.'
+                let instanceMemberNameOnly = Array.last nameSegments
+                let isSourcePropSameAsReplacementProp = List.tryFind (fun sourceInstanceMemberName -> sourceInstanceMemberName = instanceMemberNameOnly) names
+                match isSourcePropSameAsReplacementProp with
+                | Some _ ->
+                    let typeName = fullyQualifiedInstanceMember.Substring(0, fullyQualifiedInstanceMember.Length-instanceMemberNameOnly.Length-1)
```

Comment:
put spaces around the `-` operator please

Reply:
@knocte thanks. I've updated the PR with this fix.